nextflow.config

```diff
@@ -58,7 +58,7 @@ params {
 
     // Alignment
     aligner                    = 'star_salmon'
-    pseudo_aligner             = null
+    pseudo_aligner             = 'salmon'
```

Comment:
Hmmm...wondering whether this will break other stuff in the pipeline if this is on by default 🤔

Reply:
Addressed in https://github.com/nf-core/rnaseq/pull/1124/commits/cfd9270b6e42a27e3175b1635ec9233d74652e07